fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/UnnestedRecordTypeTest.java

```diff
@@ -838,4 +911,378 @@ void indexDoubleNestedType() {
             commit(context);
         }
     }
+
+    @Test
+    void deleteRecordsWhere() {
```

Comment:
Is there any need to add a test that has an `and` in the `deleteWhere` predicate? I don't see one, but I may have forgotten.